homeassistant/bootstrap.py

```diff
@@ -421,5 +428,11 @@ async def async_setup_multi_components(domains: Set[str]) -> None:
 
         await async_setup_multi_components(stage_2_domains)
 
+    # The websocket has already started.
```

Comment:
I'm not sure this is safe to do, but I haven't come up with another way to only get the startup tasks

Reply:
Didn't solve it either.

Reply:
Removing it solves it and everything appears to work as expected.  I'm concerned it might mean we fire the started event too early though.

Reply:
Not tracking the websocket tasks seems to have solved this.  I put the block back in with a timeout just to be sure.